funcions/5.js

```diff
@@ -0,0 +1,24 @@
+let text =
+	'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Distinctio necessitatibus vero voluptatum enim, maiores neque odio incidunt tempora aliquid! Nemo iusto ipsam quae quos odit soluta in nobis a doloribus?'
+
+let symbolsToRemove = /[^A-Za-z0-9]/g
+function oldWordSearch(w) {
+	var wordIsPresent = false
+	var textArr = text.toLowerCase().replace(symbolsToRemove, ' ').split(' ')
+	for (var i = 0; i < textArr.length; i++) {
+		if (textArr[i] === w) {
+			wordIsPresent = true
+		}
+	}
+	return wordIsPresent
+}
+
+function newWordSearch(w) {
+	let includes = text.toLowerCase().replace(symbolsToRemove, ' ').includes(w)
+	return includes
+}
```

Comment:
I think searching for `Lore` would return true -- the task requires a full word match.

Reply:
Checked this, returns false